src/libraries/System.Private.CoreLib/src/System/Double.cs

```diff
@@ -798,14 +808,17 @@ bool IFloatingPoint<double>.TryWriteSignificandLittleEndian(Span<byte> destinati
         /// <inheritdoc cref="IFloatingPointIeee754{TSelf}.NegativeZero" />
         static double IFloatingPointIeee754<double>.NegativeZero => NegativeZero;
 
-        /// <inheritdoc cref="IFloatingPointIeee754{TSelf}.Pi" />
-        static double IFloatingPointIeee754<double>.Pi => Pi;
-
         /// <inheritdoc cref="IFloatingPointIeee754{TSelf}.PositiveInfinity" />
         static double IFloatingPointIeee754<double>.PositiveInfinity => PositiveInfinity;
 
-        /// <inheritdoc cref="IFloatingPointIeee754{TSelf}.Tau" />
-        static double IFloatingPointIeee754<double>.Tau => Tau;
+        /// <inheritdoc cref="IFloatingPointIeee754{TSelf}.Atan2(TSelf, TSelf)" />
+        public static double Atan2(double y, double x) => Math.Atan2(y, x);
+
+        /// <inheritdoc cref="IFloatingPointIeee754{TSelf}.Atan2Pi(TSelf, TSelf)" />
+        public static double Atan2Pi(double y, double x)
+        {
+            return Atan2(y, x) / Pi;
+        }
```

Comment:
Nit: why does this method use a full body rather than an expression body like the others around it?
```suggestion
        public static double Atan2Pi(double y, double x) => Atan2(y, x) / Pi;
```

Reply:
No particular reason. I probably did it because this one will be updated to a more accurate implementation in .NET 8.